src/2dplane/GridStateSpace.cpp

```diff
@@ -16,6 +18,27 @@ bool GridStateSpace::stateValid(const Vector2f &pt) const {
     return PlaneStateSpace::stateValid(pt) && !_obstacleGrid.obstacleAt(_obstacleGrid.gridSquareForLocation(pt));
 }
 
+Vector2f GridStateSpace::intermediateState(const Vector2f &source, const Vector2f &target, float minStepSize, float maxStepSize) const {
+    cout << "ASC intermediateState" << endl;
```

Comment:
Let's either drop the printouts here or wrap them in `if (debug)` and put a `bool debug` at the top of the method.  This will keep the log cleaner when we're running.